ocrd_tool.md

```diff
@@ -10,50 +10,61 @@ services](swagger).
 
 To validate a `ocrd-tool.json` file, use `ocrd ocrd-tool /path/to/ocrd-tool.json validate`.
 
-## File parameters
+## Resource parameters
 
-To mark a parameter as expecting the address of a file, it MUST declare the
-`content-type` property as a [valid media
-type](https://www.iana.org/assignments/media-types/media-types.xhtml).
-Optionally, workflow processors can be notified that this file is potentially
-large and static (e.g. a fixed dataset or a precomputed model) and should be cached
-indefinitely after download by setting the `cacheable` property to `true`.
+To mark a parameter as expecting the address of a file or directory,
+it MUST declare the `content-type` property as a [valid media
+type](https://www.iana.org/assignments/media-types/media-types.xhtml),
+which may contain _multiple values_, formatted as a single string with comma-separated items.
 
-The filename itself, i.e. the concrete value `<fpath>` of a file parameter,
-should be resolved in the following way (with `<cwd>` representing the current 
-working directory, and `<mod>` representing the distribution directory of the module):
+Optionally, workflow processors can be notified that this file or directory
+is potentially large and static (e.g. a fixed dataset or a precomputed model),
+and should be cached indefinitely after download, by setting the `cacheable` property
+to `true`.
 
-* If `<fpath>` is an `http`/`https` URL: Download to a temporary directory (if
-  `cacheable==False`) or a semi-temporary cache directory (if `cacheable==True`)
-* If `<fpath>` is an absolute path: Use as-is.
-* If `<fpath>` is a relative path (with or without directory components):
-  Try resolving via the following directories, and return the first one found if any,
-  otherwise abort with an error message stating so:
-  * `<cwd>/<fpath>` (**Note** that the file is expected to be directly under `<cwd>`, not in a subdirectory)
-  * If an environment variable is defined that has the name of the processor in
-    upper-case and with `-` replaced with `-` and followed by `_PATH` (e.g. for a processor
-    `ocrd-dummy`, the variable would need to be called `OCRD_DUMMY_PATH`):
-    * Split the variable value at `:` and try to resolve by appending `<fpath>`
-      to each token and return the first found file if any
-  * `$XDG_DATA_HOME/ocrd-resources/<name-of-processor>/<fpath>` (with `$HOME/.local/share` instead of `$XDG_DATA_HOME` if unset)
-  * `/usr/local/share/ocrd-resources/<name-of-processor>/<fpath>`
-  * `<mod>/<fpath>` (**Note** that the file is expected to be directly under `<mod>`, not in a subdirectory)
+The path of the file or directory itself, i.e. the concrete value `<rpath>` of a 
+resource parameter, should be resolved in the following way:
+
+* If `<rpath>` is an `http:` or `https:` URI, download to a temporary directory
+  (if `cacheable==False`) or a semi-temporary cache directory (otherwise).
```

Comment:
Have we implemented this?

Reply:
see [above](https://github.com/OCR-D/spec/pull/263#issuecomment-2718528648)